plugins/junit_rt/test/com/intellij/junit4/JUnit4TestRunnerUtilTest.java

```diff
@@ -0,0 +1,150 @@
+// Copyright 2000-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
```

Comment:
Sorry for a silly question, but I wanted to make sure I am getting the license and author requirement right (for sure not the latter as of now). I think intellij generated that for me automatically when I added the file -- is that the version of the license notice? For the author name, can you please instruct me where to add that and in what format (my name vs github user, on the file vs class).

Reply:
Current copyright is
```
// Copyright 2000-2021 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
```
I don't think that we include names in the license of contributors

Reply:
Looking good then, thank you!